shared/src/errors.rs

```diff
@@ -50,6 +52,8 @@ pub enum SamplyBeamError {
     CertificateError(#[from] CertificateInvalidReason),
     #[error("Timeout executing HTTP request: {0}")]
     HttpTimeoutError(Elapsed),
+    #[error("Expired receiver public keys: {0:?}")]
```

Comment:
Is this only about expiration? What if the `to` target is not found at all because it never existed?

Reply:
Good point I will change it to `Invalid receivers:`.